imap/http_dav.c

```diff
@@ -5247,11 +5247,11 @@ int meth_get_head(struct transaction_t *txn, void *params)
     r = dav_parse_req_target(txn, gparams);
     if (r) return r;
 
-    if (txn->req_tgt.namespace->id == URL_NS_PRINCIPAL) {
```

Comment:
What does getting rid of the principal check do? Are we sure that can go (maybe it's just encompassed by not having a mbentry?)

Can TGT_SERVER_INFO somehow be set outside of a URL_NS_PRINCIPAL namespace->id, and now we're going to start intercepting requests we otherwise wouldn't?

Reply:
TGT_SERVER_INFO can not be set for any other namepsace, which is why I removed the namespace check.  It was there previously to return 204 for any other principal, but now that is captured by any URL that doesn't map to a mbentry.